x-pack/plugins/integration_assistant/common/api/categorization/categorization_route.ts

```diff
@@ -22,6 +23,7 @@ export const CategorizationRequestBody = z.object({
   packageName: PackageName,
   dataStreamName: DataStreamName,
   rawSamples: RawSamples,
+  samplesFormat: SamplesFormat.optional(),
```

Comment:
The `samplesFormat` parameter is required in the ECS call, but optional in the categorization call. Shouldn't it be required everywhere?

Reply:
I think yeah.. We can make this required everywhere...

Reply:
Added in [f0fe3ea](https://github.com/elastic/kibana/pull/191749/commits/f0fe3ea39ef3dcb6d6ec30939e4a98dec041f70a)